.github/workflows/aks-byocni.yaml

```diff
@@ -14,7 +14,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   #
-  # pull_request: {}
+  pull_request: {}
```

Comment:
Please don't forget to revert this. Thanks!

Reply:
Removed it and squashed the other two commits.